### ISSUE TITLE
Nanoc 4.2.1 compilation time more than older version 3.7.5

### ISSUE THREAD

**Issue by pmalik56** [nanoc/nanoc#878]
I have upgraded my sites from nanoc-3.7.5 to nanoc-4.2.1. And now it's taking almost 3 times more time for compilation. Don't know how to minimize the compilation time. Is it a normal behavior , I guess in upgraded version less time should be taken.

Any clue this how to minimize the compilation time.

**Comment by ddfreyne** [comment]
Hi! Can you try compiling with Nanoc 3.8 and see whether the slowdown also happens there? If so, this is likely a duplicate of #864.

**Comment by pmalik56** [comment]
Is nanoc compiling all files irrespective of changes and if yes then is there any way to stop it and compile only those file have changed or is there any way to reduce the compilation time in upgraded nanoc version.

And using command nanoc compile --verbose not getting the full description where time taken.
Is there anyway to check detail compilation time taken by each file.

![nanoc_compile](https://cloud.githubusercontent.com/assets/4602600/16446130/c57561cc-3dee-11e6-813b-3e7762c7da33.png)

**Comment by ddfreyne** [comment]
Can you share the output of `nanoc compile --debug`? It will be pretty verbose, so perhaps a gist is better.

**Comment by ddfreyne** [comment]
This is potentially related to #866.

**Comment by pmalik56** [comment]
It was taking longer time for static content compilation , which is not required and I have made some changes in config.yaml so now it's taking same or lesser time in nanoc 4.2.1. 

But I am facing one issue here when trying to compile few .less file which is inside '/static/component/...../less/*.less'  folder. 
I have the compilation rules for these less files but looks like it's not looking at the static directory for compilation. But in nanoc 3.7.5 it was working for compiling files inside static directory .

And when I am adding this below configuration in config.ymal in latest nanoc version 4.2.1 then it's compiling everything inside static which is not required.

   type: filesystem
    items_root: /static
    content_dir: 'static'
    layouts_dir: null

**Basically I need to compile few files which is inside static folder and it seems not working by defining rules may be some configuration changes required for this.**

**Comment by ddfreyne** [comment]
@pmalik56 Is this issue still related to the slowdown in Nanoc 4.2.1? I’m not sure I follow how the two things are connected.

**Comment by pmalik56** [comment]
No that slowness issue is fixed anyway it looks faster.

**Comment by ddfreyne** [comment]
@pmalik56 The [Nanoc Google Group](https://groups.google.com/forum/#!forum/nanoc) is the recommended place to get support, rather than GitHub; the latter is intended exclusively for issues.
